lib/core/src/lib/datatable/components/amount-cell/amount-cell.component.html

```diff
@@ -0,0 +1,10 @@
+<ng-container *ngIf="value$ | async as amount">
+    <span [title]="amount">
```

Comment:
WDYT about add also the following class
```suggestion
    <span [title]="amount" class="adf-amount-cell-value">
```

Reply:
same as above https://github.com/Alfresco/alfresco-ng2-components/pull/8984#discussion_r1352075874